### PR TITLE
nodeadm: block until daemon status changes are reflected

### PR DESCRIPTION
**Pull request by ndbaker1**
**Issue #, if available:**

more general attempt than https://github.com/awslabs/amazon-eks-ami/pull/1942 at solving the related issue

**Description of changes:**

revamp the retrier helper a bit and block on daemon `EnsureRunning` impls to guarantee the service is up before continuing with the rest of the bootstrap. This makes more sense for an `EnsureRunning` api, which only returns once the process is currently running, rather than just initiated.

* fixed a bug in how `GetDaemonStatus` was implemented for systemd. The current implementation was using the string value of a pointer rather than the value of the systemd property.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**

built a dev ami and verified completion of the bootstrap. also logged the `GetDaemonStatus` calls on a dev ami, and recorded 10 second transition from `activiting` to `active` for both `containerd` and `kubelet`.

This increases the total time-to-node-join metric, but would reduce potential retries in other parts of the bootstrap where connectivity to containerd or etc is required.

<details>
 <summary>nodeadm test logs</summary>

```
[root@ip-172-31-15-250 ~]# journalctl -u nodeadm-run
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal systemd[1]: Starting nodeadm-run.service - EKS Nodeadm Run...
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.7596016,"caller":"init/init.go:54","msg":"Checking user is root.."}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.7599344,"caller":"init/init.go:62","msg":"Loading configuration..","configSource":"imds://user-data"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.7624507,"caller":"init/init.go:71","msg":"Loaded configuration","config":{"metadata":{"creationTimestamp":null},"spec":{"cluster":{"name":"my-cluster","apiServerEndpoint":">
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.7627125,"caller":"init/init.go:73","msg":"Enriching configuration.."}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.762764,"caller":"init/init.go:153","msg":"Fetching instance details.."}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: SDK 2024/09/16 20:07:51 DEBUG attempting waiter request, attempt count: 1
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9515529,"caller":"init/init.go:166","msg":"Instance details populated","details":{"id":"i-0eac6143cb37792cb","region":"us-west-2","type":"m1.small","availabilityZone":"us-w>
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9515986,"caller":"init/init.go:167","msg":"Fetching default options..."}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9525166,"caller":"init/init.go:175","msg":"Default options populated","defaults":{"sandboxImage":"602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pause:3.5"}}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9526398,"caller":"init/init.go:78","msg":"Validating configuration.."}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9526534,"caller":"init/init.go:83","msg":"Creating daemon manager.."}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9537528,"caller":"init/init.go:117","msg":"Setting up system aspects..."}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9538164,"caller":"init/init.go:120","msg":"Setting up system aspect..","name":"local-disk"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.95388,"caller":"system/local_disk.go:26","msg":"Not configuring local disks!"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9539227,"caller":"init/init.go:124","msg":"Set up system aspect","name":"local-disk"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9539704,"caller":"init/init.go:120","msg":"Setting up system aspect..","name":"networking"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9540231,"caller":"system/networking.go:79","msg":"writing eks_primary_eni_only network configuration"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.965532,"caller":"init/init.go:124","msg":"Set up system aspect","name":"networking"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517271.9656186,"caller":"init/init.go:133","msg":"Ensuring daemon is running..","name":"containerd"}
Sep 16 20:07:51 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:52 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:52 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:52 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:52 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:53 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:53 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:53 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:54 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:54 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:54 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:54 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:55 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:55 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:55 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:55 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:56 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:56 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:56 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:56 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:57 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:57 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:57 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:57 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:58 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:58 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:58 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:58 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:59 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:59 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:59 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:07:59 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:00 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:00 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:00 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:00 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of containerd.service : "activating"
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517281.810569,"caller":"init/init.go:137","msg":"Daemon is running","name":"containerd"}
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517281.810625,"caller":"init/init.go:139","msg":"Running post-launch tasks..","name":"containerd"}
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517281.810633,"caller":"containerd/sandbox.go:21","msg":"Looking up current sandbox image in containerd config.."}
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517281.9384398,"caller":"containerd/sandbox.go:33","msg":"Found sandbox image","image":"602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pause:3.5"}
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517281.948726,"caller":"containerd/sandbox.go:35","msg":"Fetching ECR authorization token.."}
Sep 16 20:08:01 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517281.9796603,"caller":"containerd/sandbox.go:49","msg":"Pulling sandbox image..","image":"602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pause:3.5"}
Sep 16 20:08:02 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517282.8603086,"caller":"containerd/sandbox.go:54","msg":"Finished pulling sandbox image","image-ref":"sha256:6996f8da07bd405c6f82a549ef041deda57d1d658ec20a78584f9f436c9a3bb7"}
Sep 16 20:08:02 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517282.8603635,"caller":"init/init.go:143","msg":"Finished post-launch tasks","name":"containerd"}
Sep 16 20:08:02 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517282.8603747,"caller":"init/init.go:133","msg":"Ensuring daemon is running..","name":"kubelet"}
Sep 16 20:08:02 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:03 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:03 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:03 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:03 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:04 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:04 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:04 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:04 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:05 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:05 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:05 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:05 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:06 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:06 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:06 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:06 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:07 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:07 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:07 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:07 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:08 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:08 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:08 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:08 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:09 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:09 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:09 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:09 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:10 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:10 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:10 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:10 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:11 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:11 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:11 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:11 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:12 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:12 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:12 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:12 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:13 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:13 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:13 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: status of kubelet.service : "activating"
Sep 16 20:08:14 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517294.0026217,"caller":"init/init.go:137","msg":"Daemon is running","name":"kubelet"}
Sep 16 20:08:14 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517294.002651,"caller":"init/init.go:139","msg":"Running post-launch tasks..","name":"kubelet"}
Sep 16 20:08:14 ip-172-31-15-250.us-west-2.compute.internal nodeadm[2131]: {"level":"info","ts":1726517294.0026739,"caller":"init/init.go:143","msg":"Finished post-launch tasks","name":"kubelet"}
```
</details> 

*[See this guide for recommended testing for PRs.](../doc/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

### CODE REVIEW

**Comment by ndbaker1** [comment]
/ci
+workflow:os_distros al2023

**Comment by ndbaker1** [comment]
/ci
+workflow:os_distros al2023

File: nodeadm/internal/daemon/interface.go [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/api"
+)
 
 type Daemon interface {
 	// Configure configures the daemon.
 	Configure(*api.NodeConfig) error
-
 	// EnsureRunning ensures that the daemon is running.
 	// If the daemon is not running, it will be started.
 	// If the daemon is already running, and has been re-configured, it will be restarted.
```
Comment (cartermckinnon) [inline nodeadm/internal/daemon/interface.go:14]: update the doc to specify that it should block until running?

File: nodeadm/internal/kubelet/daemon.go [hunk 2]
```diff
@@ -44,8 +49,20 @@ func (k *kubelet) Configure(cfg *api.NodeConfig) error {
...
+	return util.NewRetrier(util.WithRetryAlways(), util.WithBackoffFixed(250*time.Millisecond)).Retry(ctx, func() error {
+		status, err := k.daemonManager.GetDaemonStatus(KubeletDaemonName)
+		if err != nil {
+			return err
+		}
+		if status != daemon.DaemonStatusRunning {
+			return fmt.Errorf("%s status is not %q", KubeletDaemonName, daemon.DaemonStatusRunning)
+		}
+		return nil
+	})
```
Comment (cartermckinnon) [inline nodeadm/internal/kubelet/daemon.go:65]: this block is identical in `containerd/daemon.go`, any reason we can't move this down into daemonmanager.StartDaemon?

File: nodeadm/internal/daemon/systemd.go [hunk 3]
```diff
@@ -55,7 +55,7 @@ func (m *systemdDaemonManager) GetDaemonStatus(name string) (DaemonStatus, error
 	if err != nil {
 		return DaemonStatusUnknown, err
 	}
-	switch status.Value.String() {
+	switch status.Value.Value().(string) {
```
Comment (cartermckinnon) [inline nodeadm/internal/daemon/systemd.go:58]: Not understanding this change. If our existing code was mishandling this, wouldn't we always get `DaemonStatusUnknown` back?

Comment (ndbaker1) [reply nodeadm/internal/daemon/systemd.go:58]: we didn't have any callers of `GetDaemonStatus` before this, so we didn't *mishandle* it, but it just never worked 🫠

File: nodeadm/internal/kubelet/daemon.go (hunk 2 above)
Comment (ndbaker1) [reply nodeadm/internal/kubelet/daemon.go:65]: im good with that if it means we change the expectation of `StartDaemon` to essentially be the same as `EnsureRunning`. ie. block until running/active

File: nodeadm/internal/daemon/interface.go (hunk 1 above)
Comment (ndbaker1) [reply nodeadm/internal/daemon/interface.go]: updated 👍

File: nodeadm/internal/kubelet/daemon.go (hunk 2 above)
Comment (ndbaker1) [reply nodeadm/internal/kubelet/daemon.go]: for better uniformity i pushed the waiter logic into the `daemonmanager` and enforced it in `Start`, `Stop`, and `Restart`

File: nodeadm/internal/daemon/systemd.go [hunk 4]
```diff
@@ -102,3 +110,19 @@ func (m *systemdDaemonManager) Close() {
 func getServiceUnitName(name string) string {
 	return fmt.Sprintf("%s.service", name)
 }
+
+func (m *systemdDaemonManager) waitForStatus(ctx context.Context, unitName string, targetStatus DaemonStatus) error {
```
Comment (cartermckinnon) [inline nodeadm/internal/daemon/systemd.go:114]: much cleaner, thx

**Review by cartermckinnon — APPROVED** [review]
LGTM. @ndbaker1 can you make sure a fresh CI run goes green before you merge?

**Comment by ndbaker1** [comment]
/ci
+workflow:os_distros al2023

**Comment by ndbaker1** [comment]
should be the last one 🤞 

/ci
+workflow:os_distros al2023

**Comment by cartermckinnon** [comment]
fyi @ndbaker1 you should be able to just use slash `ci`, it'll target al2023 if the only change is in `nodeadm` 👍

**Pull request merged**